rootfs/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk add --update --virtual .build-deps sudo build-base ruby-dev \
     &&  bundle install --gemfile=/fluentd/deis-output/Gemfile \
     &&  rake --rakefile=/fluentd/deis-output/Rakefile build \
     &&  fluent-gem install --no-document fluent-plugin-kubernetes_metadata_filter -v 2.1.4 \
-    &&  fluent-gem install --no-document fluent-plugin-elasticsearch -v 2.12.0 \
+    &&  fluent-gem install --no-document fluent-plugin-elasticsearch -v 4.0.4 \
```

Comment:
Very nice! :100: Hopefully this upgrade will work for all users... Anyway we could find all changelongs from 2.12 to 4.0.4 so we can see what has changed and if anything would break.

Reply:
https://github.com/uken/fluent-plugin-elasticsearch/blob/master/History.md

there's a changelog! They don't seem to list any kind of breaking changes or compatibility issues anywhere though